frontend/src/reducers/creditTransferReducer.js

```diff
@@ -77,6 +78,61 @@ const creditTransfers = (state = {
         success: true,
         items: action.data
       });
+    case 'RECEIVE_APPROVED_CREDIT_TRANSFERS':
+      return Object.assign({}, state, {
```

Comment:
Repeat of a question I asked last week, is there a reason for doing Object.assign here, rather than 
return {...state, etc. etc.}
Am I wrong, but I don't think you're really winning anything here using Object.assign since nothing is deep copied copied anyway?

Reply:
Hmmm, base reasoning is I was just following the previous code samples. I can give the code you suggested a try and see if it doesn't cause any problems.

Reply:
I changed some of them to use the spread operator. I'll do more changes as I refactor the reducers more.

Reply:
I just think it's easier to read - and less code (more elegant).